src/app/components/Recommendations/RecommendationsPromoImage/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useContext } from 'react';
+import { createSrcsets } from '#lib/utilities/srcSet';
+import buildIChefURL from '#lib/utilities/ichefURL';
+import Image from '#app/components/Image';
+import { Recommendation } from '#models/types/onwardJourney';
+import { RequestContext } from '#app/contexts/RequestContext';
+
+const RecommendationsImage = ({
+  image,
+  lazyLoad = false,
+}: {
+  image: Recommendation['image'];
+  lazyLoad?: boolean;
+}) => {
+  const { isAmp } = useContext(RequestContext);
+  const { height, width, altText, copyrightHolder, originCode, locator } =
```

Comment:
Apologies for not spotting this in the BFF PR, but can we have the BFF returning an ichef URL, so that we don't need to worry about building the URL in Simorgh?